### PR TITLE
Add parallel 65

### PR DESCRIPTION
**Pull request by matthewdias**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

### CODE REVIEW

File: keyboards/parallel/parallel_65/soldered/info.json [hunk 1]
```diff
@@ -0,0 +1,467 @@
+{
+    "keyboard_name": "Parallel 65% PCB",
+    "maintainer": "matthewdias",
+    "layout_aliases": {
+        "LAYOUT_65_ansi_split_bs": "LAYOUT_65_ansi_blocker_split_bs",
+        "LAYOUT_65_ansi": "LAYOUT_65_ansi_blocker",
+        "LAYOUT_65_tsangan_split_bs": "LAYOUT_65_ansi_blocker_tsangan_split_bs",
+        "LAYOUT_65_iso": "LAYOUT_65_iso_blocker"
+    },
```
Comment (fauxpark) [inline keyboards/parallel/parallel_65/soldered/info.json:9]:
```suggestion
```

File: keyboards/parallel/parallel_65/hotswap/info.json [hunk 2]
```diff
@@ -0,0 +1,159 @@
+{
+    "keyboard_name": "Parallel 65% Hotswap PCB",
+    "maintainer": "matthewdias",
+    "layout_aliases": {
+        "LAYOUT": "LAYOUT_65_ansi_blocker",
+        "LAYOUT_65_tsangan": "LAYOUT_65_ansi_blocker_tsangan"
+    },
```
Comment (fauxpark) [inline keyboards/parallel/parallel_65/hotswap/info.json:7]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/parallel/parallel_65/soldered/info.json (hunk 1 above)
Comment (matthewdias) [reply keyboards/parallel/parallel_65/soldered/info.json:9]: Confused what this change is for, will it not break the JSON format?

Comment (fauxpark) [reply keyboards/parallel/parallel_65/soldered/info.json:9]: Since this is a new board, there is no need to add layout aliases.

**Review by fauxpark — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**